README.md

```diff
@@ -6,9 +6,9 @@
 [![Tests](https://github.com/BETSRG/GHEDesigner/actions/workflows/tests.yml/badge.svg)](https://github.com/BETSRG/GHEDesigner/actions/workflows/tests.yml)
 
 ## Documentation
-[![Documentation Status](https://readthedocs.org/projects/ghedesigner/badge/?version=latest)](https://ghedesigner.readthedocs.io/en/latest/?badge=latest)
 
-Project documentation is hosted on [ReadTheDocs](https://ghedesigner.readthedocs.io/en/latest/)
+Project documentation is available at [https://github.com/BETSRG/GHEDesigner](https://github.com/BETSRG/GHEDesigner)
+
```

Comment:
I think it should link to something like this: `https://betsrg.github.io/GHEDesigner/`

Reply:
Doh! Fixed.